src/main/java/org/alliancegenome/curation_api/controllers/crud/TranscriptCrudController.java

```diff
@@ -19,19 +25,44 @@
 @RequestScoped
 public class TranscriptCrudController extends BaseEntityCrudController<TranscriptService, Transcript, TranscriptDAO> implements TranscriptCrudInterface {
 
-	@Inject
-	TranscriptService transcriptService;
-	@Inject
-	Gff3Executor gff3Executor;
+	@Inject TranscriptService transcriptService;
+
+	@Inject Gff3TranscriptExecutor gff3TranscriptExecutor;
+	@Inject Gff3TranscriptLocationExecutor gff3TranscriptLocationExecutor;
+	@Inject Gff3TranscriptGeneExecutor gff3TranscriptGeneExecutor;
+	@Inject Gff3TranscriptCDSExecutor gff3TranscriptCDSExecutor;
+	@Inject Gff3TranscriptExonExecutor gff3TranscriptExonExecutor;
 
 	@Override
 	@PostConstruct
 	protected void init() {
 		setService(transcriptService);
 	}
 
+	@Override
 	public APIResponse updateTranscripts(String dataProvider, String assembly, List<Gff3DTO> gffData) {
-		return gff3Executor.runLoadApi(dataProvider, assembly, gffData);
+		BulkLoadFileHistory history = new BulkLoadFileHistory();
```

Comment:
This looks like code very much like for the updateExons() method. Could that common code be centralized  and being called?